custom/tests.yaml

```
@@ -4829,6 +4829,57 @@ css:
       'env(--foo)');
     aspect-ratio: |-
       return bcd.testCSSProperty('aspect-ratio', '16 / 9');
+  selectors:
+    next-sibling: |-
+      return CSS.supports('selector(div + span)');
```

Since we have a wrapper function in place, let's use it (applies to all the `return CSS.supports()` custom tests):

```suggestion
      return bcd.testCSSSelector('div + span');
```